### DISCUSSION TITLE
OpenstreetMap or ArcGIS_Map ？？？

### DISCUSSION THREAD

**Discussion by huangfengye** [teslamate-org/teslamate#3924] [Polls]
I don't like the ArcGIS_Map in the Visited panel. It always tells me "Map data not yet available".....there wasn't this issue in old version
why teslamate try to use ArcGIS_Map in the last version????
![image](https://github.com/teslamate-org/teslamate/assets/26798988/78be5a97-95ff-49cb-ab6a-d68c1df5aa56)

**Comment by Amourgo** [comment]
As a Tesla owner in Chinese mainland, I urgently need to support the source data support of Baidu Maps or AutoNavi Maps.

**Reply by huangfengye** [reply]
> As a Tesla owner in Chinese mainland, I urgently need to support the source data support of Baidu Maps or AutoNavi Maps.

Both of them don't have any free API. So it is not possible

**Reply by gzjaychen** [reply]
Is it possible to use the Off-line OpenstreetMap data?
https://download.geofabrik.de/

**Reply by hipudding** [reply]
这个可以做，我这里有一个高德地图的[demo](https://github.com/hipudding/teslamate_fix_addrs/issues/7)，看是否能满足需求？
